ext/node/polyfills/module_all.ts

```diff
@@ -185,3 +186,6 @@ export default {
   worker_threads: workerThreads,
   zlib,
 } as Record<string, unknown>;
+
+internals.nodeModuleAll = moduleAll;
```

Comment:
:/ this is now always available as `Deno[Deno.internal].nodeModuleAll`

Reply:
I think this is fine though for now, I'll keep iterating on the integration between `ext/node/polyfills/` and `ext/node/01_node.js` and `ext/node/02_require.js`.